src/test/kotlin/test/tests/hashers/CodeLongevityTest.kt

```diff
@@ -138,12 +138,92 @@ class CodeLongevityTest : Spek({
                            rev4, fileName, 0,
                            lines4[1])
 
-            /* TODO(alex): the test fails
             assertCodeLine("line2",
                            rev1, fileName, 1,
                            rev3, fileName, 2,
                            lines4[2])
-            */
+        }
+
+        afterGroup {
+            testRepo.destroy()
+        }
+    }
+
+    given("'test group #2'") {
+
+        val testRepoPath = "../testrepo2"
+        val testRepo = TestRepo(testRepoPath)
+        val fileName = "test1.txt"
+
+        // t2.1: initial insertion
+        val fileContent = listOf(
+          "test1",
+          "Addition to test1.txt in middle (Commit4)",
```

Comment:
Better to use more formal contents for tests. Too random now.

Reply:
that was exact copy of your repo's changeset that shown the problem, I don't mind though to change it to anything you like, what content you would see there?

Reply:
Yes, I know. But it was in local repo not in code. Something like:
- "Line 1",
- "Line 2",
- ...